docs/virtual_machines/liveness_and_readiness_probes.md

```diff
@@ -243,3 +250,107 @@ watchdog -t 2000ms -T 4000ms /dev/watchdog
 the watchdog will send a heartbeat every two seconds to `/dev/watchdog` and
 after four seconds without a heartbeat the defined action will be executed. In
 this case a hard `poweroff`.
+
+## Defining Guest-Agent Ping Probes
+
+A Guest-Agent probes are based on qemu-guest-agent `guest-ping`.
+This will ping the guest and return an error if the guest is not
+up and running.  To easily define this on VM spec, specify `guestAgentPing: {}`
+in VM's readiness probe spec; `virt-controller` will translate this into a
+corresponding command wrapped by `virt-probe`.
+
+> Note: You can only define one of the type of probe.
```

Comment:
This is a little confusing; you can only define one type of probe? Or you can only define one instance of type `probe`?

Reply:
I have added an small clarification here. Check if its clear now. You can only define one type of these probes on each VM.